### DISCUSSION TITLE
incorrect definition of the current phase

### DISCUSSION THREAD

**Discussion by GitStudentSem** [jasonsturges/lunarphase-js#31] [Q&A]
Hello, my name is Semyon, I'm a developer from Russia.

I noticed some inconsistency in your library with general information about the lunar phase.

I am in Russia, Arkhangelsk region, this is the northern hemisphere.

Here are two screenshots:
The first is from a site that provides information about the current phase of the moon. It says that today is July 30, the phase of the moon: "waxing moon", luminosity: 92%

The second one is from your demo site, which says it's a "full moon" phase today.

Help me understand why this difference exists and how I can make the appropriate adjustments.

Thank you

![russia](https://github.com/jasonsturges/lunarphase-js/assets/69849493/7d9b16d5-aa62-4cc3-92c0-afd950de2dd7)
![moonphase](https://github.com/jasonsturges/lunarphase-js/assets/69849493/465c786d-4325-4c8a-b6dd-64d247615ac1)

**Comment by jasonsturges** [comment]
- https://github.com/jasonsturges/lunarphase-js/issues/19
- https://github.com/jasonsturges/lunarphase-js/issues/23
- https://github.com/jasonsturges/lunarphase-js/issues/24
